src/analysis/retail/priest/shadow/modules/talents/SurgeOfDarkness.tsx

```diff
@@ -0,0 +1,126 @@
+
+// Example log: /report/JAPL1zpDfN7W8wck/33-Heroic+The+Council+of+Blood+-+Kill+(5:46)/Mayrim/standard/statistics
+class SurgeOfDarkness extends Analyzer {
+  static dependencies = {
+    abilities: Abilities,
+    eventHistory: EventHistory,
+    spellUsable: SpellUsable,
+  };
+  protected abilities!: Abilities;
+  protected eventHistory!: EventHistory;
+  protected spellUsable!: SpellUsable;
+
+  procsGained = 0;
+  procsUsed = 0;
+
+  constructor(options: Options) {
+    super(options);
+    this.active = this.selectedCombatant.hasTalent(TALENTS.SURGE_OF_DARKNESS_TALENT.id);
+
+    this.addEventListener(
+      Events.applybuff.by(SELECTED_PLAYER).spell(SPELLS.SURGE_OF_DARKNESS_TALENT_BUFF),
+      this.onBuffApplied,
+    );
+
+    this.addEventListener(
+      Events.applybuffstack.by(SELECTED_PLAYER).spell(SPELLS.SURGE_OF_DARKNESS_TALENT_BUFF),
+      this.onBuffApplied,
+    );
+
+    this.addEventListener(
+      Events.removebuff.by(SELECTED_PLAYER).spell(SPELLS.SURGE_OF_DARKNESS_TALENT_BUFF),
+      this.onBuffRemoved,
+    );
+
+    this.addEventListener(
+      Events.removebuffstack.by(SELECTED_PLAYER).spell(SPELLS.SURGE_OF_DARKNESS_TALENT_BUFF),
+      this.onBuffRemoved,
+    );
+  }
+
+  onBuffApplied(event: ApplyBuffEvent | ApplyBuffStackEvent) {
+    this.procsGained += 1; // Add a proc to the counter
+  }
+
+  onBuffRemoved(event: RemoveBuffEvent | RemoveBuffStackEvent) {
+    if (
+      this.eventHistory.last(
+        1,
+        100,
+        Events.cast.by(SELECTED_PLAYER).spell(TALENTS.MIND_SPIKE_TALENT),
+      ).length === 0
+    ) {
+      // If MB is not instant, it's not a proc
+      return;
+    }
+
+    this.procsUsed += 1;
+  }
+
+  get procsWasted() {
+    return this.procsGained - this.procsUsed;
+  }
+
+  get suggestionThresholds() {
+    return {
+      actual: this.procsWasted / this.procsGained,
+      isGreaterThan: {
+        minor: 0,
+        average: 0,
+        major: 0.1,
+      },
+      style: ThresholdStyle.PERCENTAGE,
+    };
+  }
+
+  suggestions(when: When) {
+    when(this.suggestionThresholds).addSuggestion((suggest, actual, recommended) =>
+      suggest(
+        <>
+          You wasted {this.procsWasted} out of {this.procsGained}{' '}
+          <SpellLink id={TALENTS.SURGE_OF_DARKNESS_TALENT.id} /> procs.
+        </>,
+      )
+        .icon(TALENTS.SURGE_OF_DARKNESS_TALENT.icon)
+        .actual(
+          t({
+            id: 'priest.shadow.suggestions.surgeofdarkness.efficiency',
+            message: `You wasted ${this.procsWasted} out of ${this.procsGained} Surge of Darkness procs.`,
+          }),
+        )
+        .recommended(`${recommended} is recommended.`),
+    );
+  }
+
+  statistic() {
+    return (
+      <Statistic category={STATISTIC_CATEGORY.TALENTS} size="flexible">
+        <BoringSpellValueText spellId={SPELLS.SURGE_OF_DARKNESS_TALENT_BUFF.id}>
+          <>
+            {this.procsUsed}/{this.procsGained} <small>Procs Used</small>
+          </>
```

Comment:
You don't need to wrap this in a fragment as the `BoringSpellValueText` is already wrapping the contents.
```suggestion
            {this.procsUsed}/{this.procsGained} <small>Procs Used</small>
```